### DISCUSSION TITLE
Get rid of "Galaxy Brain" and "YOLO" Achievements

### DISCUSSION THREAD

**Discussion by rowan-mcalpin** [community/community#27644] [Other Feature Feedback, Questions, & Ideas] [answered]
Galaxy Brain is encouraging spam posts in this repository, and YOLO encourages contributing to open source software without other's review.

**Comment by BorislavChernev** [comment]
Click on your profile icon (top-right) -> settings -> public profile ->Profile settings -> turn off "Show achievements on my profile".

**Comment by radar-blip** [comment]
I dont think Github is going to take something away that they just handed out. They do give you the option of not showing them on your profile though (like @BorislavChernev pointed out). I also think that in the near future you will have a choice in showing/hiding individual achievement badges.

**Comment by rowan-mcalpin** [comment]
My concern isn't myself, as I can use best practices with code.

Others who don't have as much experience might not understand the goals of the achievements, and think that merging a PR without review is a good practice (when the opposite is true).

Take a look at the discussions and you can see tons of spam posts from people trying to get galaxy brain.

**Reply by jfullstackdev** [reply]
yes, it will encourage spam posts and everything just to get the badge

but what will they get ? just the badge ? can they fake the interviews ? onboarding and anything ? so it's just them just faking themselves

with so many applications I've tried, actually, I'm just interested in onboarding, the skills will really be exposed once doing, particularly the technical interview or hands-on exam

as for the badges, they make sense, as in answering valid questions ( that again will encourages spams to no benefit ) and the YOLO as in merging without a review because all agreed and confident that will work

**Comment by github-staff** [comment, marked as answer]
Thanks so much for posting in our Community @rowan-mcalpin. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
